### ISSUE TITLE
Phantom lines reported on Python 3.8

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#714]
(split off from #707)

@lhupfeldt said:

> multiconf/envs.py 210 37 82 2 86.644% 313-1149, 91->125, 92->91
> This file only has 307 lines, something has gone completely wrong.

Here's my simple reproduction:
```
$ cat -n multiline.py
     1	def f(x):
     2	    return 17
     3
     4	print(
     5	    f(
     6	        1
     7	    )
     8	)

$ venv38/bin/python3.8 -m coverage run multiline.py
17

$ venv38/bin/python3.8 -m coverage report -m
Name           Stmts   Miss  Cover   Missing
--------------------------------------------
multiline.py       5      2    60%   261-516
```

Multi-line function calls now include backwards steps in the bytecode-to-line number table, so that the execution can be attributed to the first line instead of the last.  Those negative steps are being mis-interpreted by coverage, resulting in phantom lines.

**Comment by nedbat** [comment]
@lhupfeldt I've made some fixes. Can you try installing from GitHub to see if it works for you?
```
pip install git+https://github.com/nedbat/coveragepy.git@v4.5.x#egg=coverage==0.0
```

**Comment by lhupfeldt** [comment]
Solved!

**Comment by nedbat** [comment]
Fixed in 5aca5af0 (for 5.0) and ccc92dd3 (for 4.5.x)

**Comment by nedbat** [comment]
This fix is in 4.5.2, just released: https://pypi.org/project/coverage/4.5.2/
